completions/fish/crio.fish

```diff
@@ -44,6 +44,7 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l default-sysctls -r -d 'Sys
 complete -c crio -n '__fish_crio_no_subcommand' -f -l default-transport -r -d 'A prefix to prepend to image names that cannot be pulled as-is'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l default-ulimits -r -d 'Ulimits to apply to containers by default (name=soft:hard) (default: [])'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-metrics -d 'Enable metrics endpoint for the server on localhost:9090'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-custom-shm-size -d 'Enable users to set a custom shm size instead of using the default value of 64M'
```

Comment:
If the default size ever changes, we'll have to make sure to remove it here too, a likely place to forget.
```suggestion
complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-custom-shm-size -d 'Enable users to set a custom shm size instead of using the default value'
```

Reply:
OK, I see the 64M is everywhere else, so less likely this would be missed, Ignore this one.